### PR TITLE
Support for release tags

### PR DESCRIPTION
**Pull request by marayl**
The Docker compose file should have variable placeholders for the image tag name.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 version: "3"
 services:
   mariadb:
-    image: registry.gitlab.com/reactivemarkets/private-registry/mariadb:beta
+    image: registry.gitlab.com/reactivemarkets/private-registry/mariadb:${REACTIVE_TAG}
```
Comment (markmcdowell) [inline docker-compose.yml]: I think doing this complicates it, ideally we'd remove the .env completely once a docker build with a default password is built.

We may need to update the docker-compose when we move major versions but that's better for the end user.

Comment (marayl) [reply docker-compose.yml]: Okay. I will put the version directly in the docker-compose file.

Comment (markmcdowell) [reply docker-compose.yml]: actually you could keep the .env but check it in

Comment (marayl) [reply docker-compose.yml]: Ah, missed this. I will update in my next PR.

**Pull request merged**